### PR TITLE
Add ENV variable for custom chia arguments

### PR DESCRIPTION
**Pull request by carles2m**
For example, to be able to set --passphrase-file, without which chia will not start or start corrupting an existing blockchain db.

### CODE REVIEW

**Comment by carles2m** [comment]
> Hey there! Thanks for the submission. Would you please make one additional change of adding this new var to the list of ENV declarations in the Dockerfile? We like to have all the available variables that can be set here: https://github.com/Chia-Network/chia-docker/blob/main/Dockerfile#L25
> 
> Just looking for a new line with `ENV chia_args=`

Makes total sense, added!

**Comment by carles2m** [comment]
BTW let me know if you'd like me to make some update to the readme file to document this variable.

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ cd /chia-blockchain || exit 1
 # shellcheck disable=SC1091
 . ./activate
 
-chia init --fix-ssl-permissions
+chia ${chia_args} init --fix-ssl-permissions
```
Comment (Starttoaster) [inline docker-entrypoint.sh:14]:
```suggestion
# shellcheck disable=SC2086
chia ${chia_args} init --fix-ssl-permissions
```

**Review by Starttoaster — APPROVED** [review]
LGTM

**Pull request merged**